support/behandeling-exporter.js

```diff
@@ -48,22 +48,49 @@ function createBehandelingExtract(zitting, agendapunt, isWrappedInZittingInfo =
   } else {
     behandelingHTML = generatePrivateBehandelingHTML(agendapunt);
   }
+  const errors = [];
+  const behandelingErrors = validateBehandeling(agendapunt);
+  errors.push(...behandelingErrors);
   if (isWrappedInZittingInfo) {
-    return wrapZittingInfo(zitting, behandelingHTML);
+    const zittingErrors = validateMeeting({
+      plannedStart: zitting.geplandeStart,
+      startedAt: zitting.start,
+      endedAt: zitting.end,
+    });
+    
+    errors.push(...zittingErrors);
+    return {html: wrapZittingInfo(zitting, behandelingHTML), errors};
   } else {
-    return behandelingHTML;
+    return {html: behandelingHTML, errors};
   }
 }
 
 function wrapZittingInfo(zitting, behandelingHTML) {
   const template = PUBLISHER_TEMPLATES.get("treatment");
   const html = template({behandelingHTML, zitting, prefixes: prefixes.join(" ")});
-  const errors = validateMeeting({
-    plannedStart: zitting.geplandeStart,
-    startedAt: zitting.start,
-    endedAt: zitting.end
-  });
-  return {html, errors};
+  return html;
+}
+
+function validateBehandeling(agendapunt) {
```

Comment:
I think it would be nice to extract this into it's own file, this helps for later refactors

Reply:
Also why not use the abstractions already available and reuse
`Decision.extractDecisionsFromDocument()` or `Decision.fromTriples` ?